![logo](https://callysto.ca/wp-content/uploads/2018/06/Callysto_Notebook-Banner_Top_06.06.18.jpg)

# Number - Solve Problems With Decimal Numbers

In Python there are four different types of numbers, but we'll only be using two of them:

**int** means integer, a positive or negative whole number e.g. 97 or -3

**float** is a floating point number, a number that has a decimal point e.g. 2.7 or -55.81

Let's use Python to solve some problems with decimal (float) numbers.

If there are 30 cupcakes divided equally among 24 students in a class, how many cupcakes would each student get?

In [24]:
30/24

1.25

Each student would get 1.25 cupcakes. That is 1 and 25 hundredths cupcakes.

If a tennis ball has a mass of 58.5 grams then what is the mass, in grams, of 3 tennis balls?

In [25]:
58.5*3

175.5

## Practice Problem 1
If you had 11 dollars and 25 cents but you spent 3 dollars, then how much do you have left?

## Practice Problem 2
If you have \$5 to split evenly amoung 3 people, about how much will each person get?

We can also round decimals using a function called *round* in a Python library called *decimal*. The following code will print the quotient of five divided by three, then print the same answer rounded to two decimal places.

In [26]:
from decimal import Decimal
answer = Decimal(5/3)
print(answer)
print(round(answer,2))

1.6666666666666667406815349750104360282421112060546875
1.67


## Practice Problem 3
What is 17 divided by 6, rounded to the nearest thousandth?